frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -476,7 +477,7 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                 ) : (
                     <>
                         {featureFlagLoading ? (
-                            <Skeleton active />
+                            <LemonSkeleton active />
```

Comment:
This change will produce a single line e.g.

|Before|After|
|----|----|
| <img width="759" alt="Screenshot 2023-12-14 at 15 10 48" src="https://github.com/PostHog/posthog/assets/6685876/b10567c6-e76c-4fef-a267-29384d821de6"> | <img width="758" alt="Screenshot 2023-12-14 at 15 11 17" src="https://github.com/PostHog/posthog/assets/6685876/5e122a0d-cc99-4afb-b617-fba687a8c67a"> |

What you need is something more like this:

```suggestion
                            <div className="space-y-2">
                                <LemonSkeleton active className="w-1/3 h-4" />
                                <LemonSkeleton active />
                                <LemonSkeleton active />
                                <LemonSkeleton active className="w-2/3 h-4" />
                            </div>
```

Which produces this empty state
<img width="764" alt="Screenshot 2023-12-14 at 15 13 29" src="https://github.com/PostHog/posthog/assets/6685876/af298c8c-daf4-4fce-9595-2943da016871">